test/calculator_test.js

```diff
@@ -0,0 +1,44 @@
+const assert = require('assert');
+const chai = require('chai');
+const calc = require('../calculator')
+
+describe('calculator', function () {
+  describe('#calculate()', function () {
+    it('adds one number', function () {
+      const calculator = calc.create_calculator('+');
+
+      calculator.add_number(16);
+
+      assert.equal(16, calculator.calculate());
+    });
+
+    it('adds two numbers', function () {
+      const calculator = calc.create_calculator('+');
+
+      calculator.add_number(5);
+      calculator.add_number(4);
+
+      assert.equal(9, calculator.calculate());
+    });
+
+    it('adds many numbers', function () {
+      const calculator = calc.create_calculator('+');
```

Comment:
ooh.. it looks like this line is repeated in each test.. maybe this could be in a beforeEach?